tutorials/algorithms/05_qaoa.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<p>Not sure if it makes much sense to have <code>brute_force</code> as a function when it admits no arguments (so it can't be used for any other problem) nor it's used again in the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>